bin/pretty.js

```diff
@@ -0,0 +1,226 @@
+#! /usr/bin/env node
```

Comment:
Ditto

Reply:
see above - is dev workstation only script and opt-in for now